code/game/objects/items/storage/belt.dm

```diff
@@ -739,7 +739,8 @@
 		/obj/item/clothing/gloves,
 		/obj/item/melee/flyswatter,
 		/obj/item/assembly/mousetrap,
-		/obj/item/paint/paint_remover
+		/obj/item/paint/paint_remover,
+		/obj/item/twohanded/broom
```

Comment:
broom is a bit big to fit on a belt

Reply:
the original intention from what i read is that its supposed to be telescopic but it doesn't inherit any of those properties like from telescopic baton so I might remove it.